tests/functional/setups/manu_thermoporomech_nofrac_2d.py

```diff
@@ -1031,16 +1036,16 @@ class ManuThermoPoroMechSolutionStrategy2d(
 ):
     """Solution strategy for the verification setup."""
 
+    exact_sol: ManuThermoPoroMechExactSolution2d
+    """Exact solution object."""
+
+    results: list[ManuThermoPoroMechSaveData] = []
```

Comment:
Isn't this initialization superfluous, since it also happens in SolutionStrategy.__init__?

Reply:
You are right, removed it.